### PR TITLE
Fix Save Layout Client11+

### PR DESCRIPTION
**Pull request by hygorrasec**
This update fixes the problem that when the player logout, the layout adjusted by him is not saved.

### CODE REVIEW

**Comment by miezaF** [comment]
Server can be bugged with this solution.
Also it would bug with over than 10.000 players on db.

File: src/player.cpp [hunk 1]
```diff
@@ -48,7 +48,7 @@ extern Imbuements* g_imbuements;
 
 MuteCountMap Player::muteCountMap;
 
-uint32_t Player::playerAutoID = 0x10000000;
+uint32_t Player::playerAutoID = 0x10010000;
```
Comment (FakeShinoda) [inline src/player.cpp:51]: so you can olny have 10.000 players ?

**Comment by hygorrasec** [comment]
> Server can be bugged with this solution.
> Also it would bug with over than 10.000 players on db.

Hi, I have not found another solution though 10k+ players online is almost impossible. The problem is just this?

**Comment by FakeShinoda** [comment]
In this case "playerAutoID" will be the same forever ?

**Comment by hygorrasec** [comment]
> In this case "playerAutoID" will be the same forever ?

I noticed that with this playerAutoID he was creating new ID's in the /characterdata folder every time the same char entered. So it dont save the layout within the game.

**Comment by gerotib** [comment]
Thats what I commented in #235, and yeah it works, but needs improvements as you guys realised. Hope we can find actually good and bug-free solution ;)

**Comment by hygorrasec** [comment]
> Thats what I commented in #235, and yeah it works, but needs improvements as you guys realised. Hope we can find actually good and bug-free solution ;)

Hey! As I dont find any PR, I did so to try solve this problem. Sorry for not seeing your issue before, this solution I found in a forum, adapted it and it worked.

**Comment by Raphaellb** [comment]
I tested this yesterday with latest build, and it is working.

**Comment by majestyotbr** [comment]
Tested, working. Thanks for the contribution.

**Pull request merged**